scripts/verify_genproto.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+# This scripts is automatically run by CI to prevent pull requests missing running genproto.sh
+# after changing *.proto file.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+./scripts/genproto.sh
+diff=$(git diff --name-only | grep -c ".pb.")
+if [ "$diff" -eq 0 ]; then
+  echo "PASSED genproto-verification!"
+  exit 0
+fi
+echo "Failed genproto-verification!" >&2
+echo "* Found changed files $(git diff --name-only | grep '.pb.')" >&2
+echo "* Please rerun genproto.sh after changing *.proto file" >&2
+echo "* Run ./scripts/genproto.sh" >&2
```

Comment:
This recommendation only makes sense when command is run in CI. 
When running locally user needs to just commit the changes.

I think we should explain that user should run `./scripts/genproto.sh` only if the commend was run in CI and then commit the changes.